actionpack/lib/action_controller/test_case.rb

```diff
@@ -581,7 +581,7 @@ def setup_request(controller_class_name, action, parameters, session, flash, xhr
         end
 
         def wrap_execution(&block)
-          if executor_around_each_request && defined?(Rails.application) && Rails.application
+          if ActionController::TestCase.executor_around_each_request && defined?(Rails.application) && Rails.application
```

Comment:
We should replace the `class_attribute` by a `singleton_class.attr_accessor`, there's no point having the extra inheritance code and instance accessor if we don't use it.

Reply:
Thanks, updated.